src/modules/previewpane/common/handlers/PreviewHandlerBase.cs

```diff
@@ -201,7 +201,7 @@ private static void RegisterPreviewHandler(string name, string extensions, strin
             }
 
             // Add preview handler to preview handler list
-            using (RegistryKey handlersKey = Registry.CurrentUser.OpenSubKey("SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\PreviewHandlers", true))
+            using (RegistryKey handlersKey = Registry.LocalMachine.OpenSubKey("SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\PreviewHandlers", true))

```

Comment:
Any reason for switching from HKCU to HKLM?

Reply:
Changed back to HKCU reason for the change was that the registration logic was failing because there is no existing `PreviewHandlers` key under current user. Updated the implementation to create a Key if not exist before